### PR TITLE
Fixing nginx

### PR DESCRIPTION
**Pull request by dharness**
I added a simple 404 to the server and moved some pipeline code into separate shell scripts for ease of use

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -15,13 +15,18 @@ http {
...
+          listen 9000;
+          server_name ~^(.+)$;
+          return 404;
+        }
+
+        server {
+          listen 9000 default_server;
           listen       443 ssl;
-          server_name  ~^(dev\.)?api.bablot.co;
+          server_name  dev.api.bablot.co 162.243.218.96;
```
Comment (moskalyk) [inline nginx/nginx.conf]: do we not need the regex prefix to api.bablot.co?

Comment (dharness) [inline nginx/nginx.conf]: Until we set up a subdomain on namecheap, we can't call it by name in production

**Comment by moskalyk** [comment]
LGTM

File: nginx/nginx.conf (hunk 1 above)
Comment (moskalyk) [inline nginx/nginx.conf]: Ahh okay this is what you were saying last night. Got it.

**Pull request merged**